packages/perseus/src/components/__tests__/zoomable_test.jsx

```diff
@@ -133,7 +133,7 @@ describe("Zoomable", () => {
         expect(container).toMatchInlineSnapshot(`
             <div>
               <span
-                style="display: block; width: 100%; transform: scale(0.4993757802746567, 0.4993757802746567); transform-origin: 0 0; opacity: 1; height: 101px; transition-property: opacity transform; transition-duration: 0.3s; transition-timing-function: ease-out;"
+                style="display: block; width: 100%; transform: scale(0.4993757802746567, 0.4993757802746567)  translate(0, 8px); transform-origin: 0 0; opacity: 0; height: 101px;"
```

Comment:
Why did this change?

Reply:
Turns out the test needed a second call to `jest.runOnlyPendingTimers()`. I'm not sure why it worked in the webapp repo... I suspect a subtle difference in the jest setup?